chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/CertificationIdController.java

```diff
@@ -125,6 +128,7 @@
 			+ "Returns a list of basic product information, " 
 			+ "Criteria and CQM calculations, and the associated CMS EHR Certification ID if one exists.")
 	@RequestMapping(value="/create", method=RequestMethod.POST, consumes=MediaType.APPLICATION_JSON_VALUE, produces={MediaType.APPLICATION_JSON_VALUE})
+	@CacheEvict("certificationIds")
```

Comment:
This is the line that refreshes the cache when/if someone creates a new CMS ID, right? Wanted to make sure this was in here